### PR TITLE
Show firmware version in web UI

### PR DESCRIPTION
**Pull request by npunniyamoorthy**
*Show firmware version in web UI
*Log firmware version

### CODE REVIEW

File: web/src/pages/Settings/About/About.jsx [hunk 1]
```diff
@@ -24,6 +33,10 @@ const About = ({ onClose }) => {
                 <br />
         Version: {info.version}
                 <br />
+        Main Unit Firmware Version: {info.fw.length ? Number(info.fw[0].version) : "No Preamp"} 
+                <br />
+        Expansion Unit Firmware Versions: {info.fw.length > 1 ? generateBoardVersionsString(info.fw): "No Expansion Units"}
```
Comment (rtertiaer) [inline web/src/pages/Settings/About/About.jsx:38]: could you conditionally display this line? an example nearby is the `info.access_key` bit below.

Comment (npunniyamoorthy) [reply web/src/pages/Settings/About/About.jsx:38]: Would you like me to do so for both lines or just the expansion unit line?

Comment (npunniyamoorthy) [reply web/src/pages/Settings/About/About.jsx:38]: nvm lincoln explained what yall wanted me to do

File: amplipi/ctrl.py [hunk 2]
```diff
@@ -236,6 +236,17 @@ def reinit(self, settings: models.AppSettings = models.AppSettings(), change_not
...
+    if self.status.info.fw:
+      fw_info = self.status.info.fw[0]
+      logger.info(f"Main unit firmware version: {fw_info.version}")
+    else:
+      logger.info(f"No preamp")
+    if self.status.info.fw[1:]:
+      for i in range(1,len(self.status.info.fw)):
+          fw_info = self.status.info.fw[i]
+          logger.info(f"Expansion unit {i} firmware version: {fw_info.version}")
+    else:
```
Comment (linknum23) [inline amplipi/ctrl.py:248]: A potentially cleaner version
```suggestion
    for i, fw_info in enumerate(self.status.info.fw[1:], start=1):
      logger.info(f"Expansion unit {i} firmware version: {fw_info..version}")
    else:
```

File: amplipi/rt.py [hunk 3]
```diff
@@ -509,7 +509,7 @@ def reset(self):
 
   def read_versions(self) -> List[Tuple[int, int, int, bool]]:
     """ Read the firmware information for all preamps (major, minor, hash, dirty?)"""
-    return []
+    return [(1, 6, 0xde0f8eb, False), (1, 7, 0xde0f8eb, False)]
```
Comment (linknum23) [inline amplipi/rt.py:512]: This fake version should probably be removed, right?

**Comment by linknum23** [comment]
Can you rebase this?

**Review by linknum23 — APPROVED** [review]
Looks good after you fix the conflicts.

**Pull request merged**